.github/workflows/preview_comment.yml

```
@@ -0,0 +1,27 @@
+name: Comment preview environment URL
+
+on:
+  pull_request:
+    types: [opened]
```

```suggestion
on:
  pull_request:
    types:
      - opened
```
好みもあるんですが他のworkflowもこっちの方が使われてたので